packages/@react-aria/slider/src/useSliderThumb.ts

```diff
@@ -237,10 +237,10 @@ export function useSliderThumb(
     inputProps: mergeProps(focusableProps, fieldProps, {
       type: 'range',
       tabIndex: !isDisabled ? 0 : undefined,
-      min: state.getThumbMinValue(index),
-      max: state.getThumbMaxValue(index),
+      min: isNaN(state.getThumbMinValue(index)) ? '' : state.getThumbMinValue(index),
```

Comment:
I'm not sure empty is the right thing to do. There should always be a min value for an input type="range". The default is described here https://html.spec.whatwg.org/multipage/input.html#range-state-(type=range)
However, I think this should be set in the state. I'm a little confused as to how to trigger the issue you've reported though. It just says, "The example from the docs page is sufficient for this". Which docs page is it, which example, and what steps do I need to do to reproduce it?

> The [min](https://html.spec.whatwg.org/multipage/input.html#attr-input-min) attribute, if specified, must have a value that is a [valid floating-point number](https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#valid-floating-point-number). The [default minimum](https://html.spec.whatwg.org/multipage/input.html#concept-input-min-default) is 0. The [max](https://html.spec.whatwg.org/multipage/input.html#attr-input-max) attribute, if specified, must have a value that is a [valid floating-point number](https://html.spec.whatwg.org/multipage/common-microsyntaxes.html#valid-floating-point-number). The [default maximum](https://html.spec.whatwg.org/multipage/input.html#concept-input-max-default) is 100.

Reply:
@snowystinger I created a reproduction repository here:
https://github.com/konsalex/react-aria-slider-NaN-reproduce

To repro:
```
yarn install
yarn test
```

You will see something along the lines.
```
stderr | src/slider.test.jsx > @neo4j-ndl/slider > range slider with steps
Warning: Received NaN for the `max` attribute. If this is expected, cast the value to a string.
```

Some code is arbitrary as I copied some parts from our component library.

Tell me if you see some red flags, that shouldn't be there and end up messing the hook.

@reidbarber could you check also the repro? It seems hard to reproduce this with only the `renderHook` utility, I would need to create something additional I guess?

Reply:
O, I see the problem. You're firing an event that is missing information. It shouldn't happen in the real world, but it can happen in a test environment pretty easily. If you're seeing it in the real world, let us know, because that'd still be a bug.
You can always look through our tests to see how we do things as well.

```
        let [thumb, rightSlider] = getAllByRole('slider');
        if (!thumb) throw new Error('Thumb not found');
        await userEvent.hover(thumb);

        fireEvent.change(thumb, {target: {value: '30'}}); // you just need to fire an event like this instead of await click
```

Reply:
@snowystinger thanks for diving to the issue!

I am thinking shouldn't this be guarded in the lib in any case? We cannot expect consumers that test to know exactly how to test a slider, feels counter-intuitive to test like this, when you want to test a simple flow like:

1. Click thumb to focus (press `tab` has other issues)
2. Press arrow keys to increase/decrease value